src/components/ApiInfo/ApiInfo.tsx

```diff
@@ -44,22 +44,20 @@ export class ApiInfo extends React.Component<ApiInfoProps> {
       null;
 
     const website =
-      (info.contact &&
-        info.contact.url && (
-          <InfoSpan>
-            URL: <a href={info.contact.url}>{info.contact.url}</a>
-          </InfoSpan>
-        )) ||
+      (info.contact && info.contact.url && (
```

Comment:
What is the reason for this change?

Reply:
I believe this is because of prettier update which improved formatting a bit